doc/content/lorawan/regional-parameters/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Regional Parameters
```

Comment:
this document could use some graphics, like showing the channel layout over the frequencies